controllers/cloud.redhat.com/clowdjobinvocation_controller.go

```diff
@@ -65,23 +77,23 @@ func (r *ClowdJobInvocationReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 	ctx = context.WithValue(ctx, errors.ClowdKey("recorder"), &r.Recorder)
 
 	cji := crd.ClowdJobInvocation{}
-	err := r.Client.Get(ctx, req.NamespacedName, &cji)
-	if err != nil {
+	if err := r.Client.Get(ctx, req.NamespacedName, &cji); err != nil {
 		if k8serr.IsNotFound(err) {
 			// Must have been deleted
 			return ctrl.Result{}, nil
 		}
-		r.Log.Error(err, "CJI not found ")
+		r.Log.Error(err, "CJI not found", "clowdjobinvocation")
```

Comment:
This will trigger a warning, because arguments 3+ are supposed to be key/value pairs, like 
```r.Log.Error(err, "message", "controller", "cji")```